Core/DotLanches.Domain/ValueObjects/Cpf.cs

```diff
@@ -0,0 +1,53 @@
+﻿using DotLanches.Domain.Exceptions;
+using System.Text.RegularExpressions;
+
+namespace DotLanches.Domain.ValueObjects
+{
+    public class Cpf
+    {
+        public string Number { get; private set; }
+
+        public Cpf(string number)
+        {
+            var cpf = ValidateAndFormatCpf(number);
+            Number = number;
```

Comment:
aqui não deveria receber o valor depois de formatar?
```suggestion
            Number = cpf;
```

Reply:
Desse jeito ele vai manter a pontuação do valor caso chegue assim:
![image](https://github.com/lanchesjaamp/dotlanche/assets/32016451/347dc96c-25f8-4c74-9fab-79e8e4705ba4)